server/hpccloud/server/models/project.py

```diff
@@ -154,7 +154,7 @@ def delete(self, user, project):
 
         super(Project, self).remove(project)
 
-    def share(self, sharer, project, users, groups):
+    def share(self, sharer, project, users, groups, flags=[]):
         """
```

Comment:
@cjh1 Is this an ok way to assign flags to the group? I'm passing `['execute']` when the project is shared with the project's group